tests/lib.rs

```diff
@@ -63,10 +63,19 @@ struct Create {
   mnemonic: Mnemonic,
 }
 
-fn create_wallet(rpc_server: &test_bitcoincore_rpc::Handle) {
-  CommandBuilder::new(format!("--chain {} wallet create", rpc_server.network()))
+fn create_wallet(rpc_server: &test_bitcoincore_rpc::Handle, wallet_name: Option<String>) {
```

Comment:
Can we always use the name ord? Is the old test actually testing anything? If not, it can be removed.